src/Illuminate/Foundation/Http/Middleware/VerifyPostSize.php

```diff
@@ -0,0 +1,53 @@
+<?php namespace Illuminate\Foundation\Http\Middleware;
+
+use Closure;
+use Illuminate\Contracts\Routing\Middleware;
+use Illuminate\Http\Exception\PostTooLargeException;
```

Comment:
Its not for the `HttpException` for the CSRF check - so I dont think so. That is where I got the coding style from: https://github.com/laravel/framework/blob/5.0/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php